subcommand/server-acl-init/rules.go

```diff
@@ -7,10 +7,13 @@ import (
 )
 
 type rulesData struct {
-	EnableNamespaces               bool
```

Comment:
I renamed these a bit:
* Put Sync or Inject at front of variable name
* Renamed "Namespace" to NS for brevity for the longer names
* Changed `K8SNSMirroring` to `NSMirroring` since it's clear we're mirroring k8s namespaces (what other namespaces would we mirror)